test/unit/deprecated/iPgmUnit.js

```diff
@@ -196,6 +226,10 @@ describe('iPgm Class Unit Tests', function () {
         });
 
       pgm.addReturn('0', '20A');
```

Comment:
[iPgm.addReturn](https://github.com/IBM/nodejs-itoolkit/blob/master/lib/Deprecated.js#L132) does not emit a deprecation warning. It should emit 

```
As of v1.0, 'iPgm.addReturn()' is deprecated. Please use 'ProgramCall.addReturn()' instead.
```

Like was done for [iPgm.addParam](https://github.com/IBM/nodejs-itoolkit/blob/master/lib/Deprecated.js#L120)